packages/main-nav/src/MainNavLink.tsx

```diff
@@ -91,9 +91,10 @@ export const MainNavButton = ({
 		<MainNavItem
 			as="button"
 			css={{
-				backgroundColor: 'transparent',
-				border: 'none',
 				appearance: 'none',
+				background: 'transparent',
+				cursor: 'pointer',
+				textAlign: 'left',
```

Comment:
textAlign should also be apply, technically it won't really do anything but it further aligns the styles with `MainNavLink`